contracts/test/TestEncryptedERC20Mintable.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause-Clear
+pragma solidity ^0.8.24;
+
+import "fhevm/lib/TFHE.sol";
+import { EncryptedERC20 } from "../token/ERC20/EncryptedERC20.sol";
+
+/**
+ * @title       TestEncryptedERC20Mintable
+ * @notice      This test contract inherits EncryptedERC20.
+ * @dev         It allows any account to mint tokens. Mint amounts are public.
+ */
+contract TestEncryptedERC20Mintable is EncryptedERC20 {
```

Comment:
Can you please reput this contract in an extension folder like before, and keep its original name which was EncryptedERC20Mintable if not mistaken?